pkg/extension/src/scripts/content/prepare-content.js

```diff
@@ -136,7 +136,15 @@
     document.documentElement.appendChild(contentCopyEl);
 
     Array.from(contentCopyEl.getElementsByTagName('*')).forEach(prepareContentPostItem);
-
+    // check if create_time is defined
```

Comment:
We probably should do this in an exception handler, just in case the value is defined to something that isn't a date.

Reply:
I wrapped the function in an exception handler @jacksonh